src/gui/static/karma.conf.js

```diff
@@ -4,12 +4,13 @@
 module.exports = function (config) {
   config.set({
     basePath: '',
-    frameworks: ['jasmine', '@angular/cli'],
+    frameworks: ['jasmine', '@angular/cli', 'sinon'],
```

Comment:
__note:__ sinon is a great library, and I believe you'll need it in a lot of GUI tests eventually.